src/YagrCore/plugins/tooltip/utils.ts

```diff
@@ -8,3 +10,65 @@ export function escapeHTML(html: string) {
     elem.innerText = html;
     return elem.innerHTML;
 }
+
+type FindClosestAreaOptions = {
+    /** cursor X */
+    x: number;
+    /** cursor Y */
+    y: number;
+    series: AlignedData;
+};
+
+export const findClosestLines = ({x, y, series}: FindClosestAreaOptions) => {
```

Comment:
Can you add unit tests for this utils, mostly to ensure valid calculation of closes area/line between two X coordinates?